apis/apps/v1alpha1/clusterdefinition_types.go

```diff
@@ -357,6 +357,11 @@ type ClusterComponentDefinition struct {
 	// +listMapKey=key
 	// +optional
 	CustomLabelSpecs []CustomLabelSpec `json:"customLabelSpecs,omitempty"`
+
+	// switchoverSpec defines command to do switchover.
+	// in particular, when workloadType=Replication, the command defined in switchoverSpec will only be executed under the condition of cluster.componentSpecs[x].SwitchPolicy.type=Noop.
+	// +optional
+	SwitchoverSpec *SwitchoverSpec `json:"switchoverSpec,omitempty"`
```

Comment:
expect updates to `spec.replicationSpec`, plz revise it.

Reply:
> expect updates to `spec.replicationSpec`, plz revise it.

i have removed replicationSpec switchPolicy and switchCmdExecuteConfig